my_list.rb

```diff
@@ -0,0 +1,34 @@
+require_relative './enumerable'
+
+# Create a class MyList that has an instance variable @list.
+class MyList
+  include Enumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  #  method #each that yields successive members of @list
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] Inside `each` method you have to implement your logic to to behave like each. kindly implement this.

Like this:

```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```